notification/resources/payment-interface-interaction-type.json

```diff
@@ -0,0 +1,55 @@
+{
+  "key": "ctp-adyen-integration-interaction-notification",
+  "name": {
+    "en": "ctp-adyen-integration-interaction-notification"
+  },
+  "resourceTypeIds": [
+    "payment-interface-interaction"
+  ],
+  "fieldDefinitions": [
+    {
+      "name": "timestamp",
+      "label": {
+        "en": "timestamp"
+      },
+      "required": true,
+      "type": {
+        "name": "String"
```

Comment:
There is also date/time type. Is String on purpose here?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/44/commits/64192281d8717eb9cf7b7097794a3cdbe6c0038a